tests/usbus/usbdev_mock.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "embUnit.h"
+#include "periph/usbdev.h"
+#include "usbdev_mock.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+static usbdev_mock_t usbdev_mock;
+static uint8_t _in_buf[256];    /* "host" in */
+static uint8_t _out_buf[64];    /* "host" out */
+
+static const usbdev_driver_t testdriver;
+
+static usbdev_mock_t *_ep2dev(usbdev_ep_t *ep)
+{
+    return (usbdev_mock_t *)ep->dev;
+}
+
+void usbdev_init_lowlevel(void)
+{}
+
+usbdev_t *usbdev_get_ctx(unsigned num)
+{
+    (void)num;
+    return &usbdev_mock.usbdev;
+}
+
+void usbdev_mock_setup(usbdev_mock_esr_cb_t esr_cb,
+                       usbdev_mock_ep_esr_cb_t ep_esr_cb,
+                       usbdev_mock_ready_cb_t ready_cb)
+{
+    memset(&usbdev_mock, 0, sizeof(usbdev_mock));
+    usbdev_mock.usbdev.driver = &testdriver;
+
+    usbdev_mock.esr_cb = esr_cb;
+    usbdev_mock.ep_esr_cb = ep_esr_cb;
+    usbdev_mock.ready_cb = ready_cb;
+}
+
+static void _init(usbdev_t *usbdev)
+{
+    usbdev_mock_t *dev = (usbdev_mock_t *)usbdev;
+
+    /* Init fake usbdev */
+
```

Comment:
Missing call or forgot to remove comment ?

Reply:
Removed